apps/common/controllers/add-another-address-loop.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const _ = require('lodash');
+const BaseController = require('./base');
+
+module.exports = class AddAnotherAddressController extends BaseController {
+  locals(req, res, callback) {
+    const locals = super.locals(req, res, callback);
+    const field = locals.field;
```

Comment:
what happens if step.locals.field is undefined? We should probably throw an informative error here to help out future-us :)